packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -118,9 +118,13 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
           items: newProps.selectedItems
         },
         () => {
-          // Need to reset focus in the same that way that we do if an item is selected by a non-controlled component
-          // See _onSelectedItemsUpdated.
-          this.resetFocus(focusIndex);
+          // Only update the focus if this component is currently focused. If the items are managed by another component,
```

Comment:
The second sentence confused me a little... 

> If the items are managed by another component, then that component should control focus

Is this also documented elsewhere rather than just having this internal comment? Do we even need to this second part of the comment?

Reply:
We might not need it. I mainly put it there as an explanation. I'll make that more clear since it's really that we don't want it stealing focus when it shouldn't.

Reply:
Fixed